pkg/ingester/user_tsdb.go

```diff
@@ -292,15 +201,15 @@ func (u *userTSDB) compactHead(blockDuration int64) error {
 	return u.db.CompactOOOHead()
 }
 
-func (u *userTSDB) persistentSeriesCallback() tsdb.SeriesLifecycleCallback {
+func (u *userTSDB) seriesLifecycleCallback() tsdb.SeriesLifecycleCallback {
 	return seriesLifecycleCallback{
```

Comment:
Can we completely remove the `seriesLifecycleCallback` now, by just exporting the methods on the userTSDB? I think this abstraction was introduced for ephemeral tsdb, but it doesn't add value anymore.

Reply:
Sure.